README.md

```diff
@@ -1,2 +1,27 @@
 # verkeersvergunningen
-A proxy-api which exposes vergunningen by querying the decos api
+A proxy-api which exposes vergunningen by querying the decos api. The focus for now is exposing "zwaar verkeer" permits 
+to the Cleopatra system, but this may change in the future to also include exposing more permits to other third parties.
+
+### Getting started
+
+1. Make a new virtualenv
+2. Install requirements: `make install`
+3. Run tests: `make test`
+
```

Comment:
Omschrijving toevoegen dat we algemene architectuur toevoegen zoals hier beschreven: https://github.com/Amsterdam/opdrachten_team_dev/tree/master/project_architecture